hawkbit-cache-redis/src/test/java/org/eclipse/hawkbit/cache/eventbus/EventDistributorTest.java

```diff
@@ -56,7 +56,7 @@ public void before() {
     @Test
     public void distributeDistributedEventSendsToRedis() {
```

Comment:
This code has to be re-think when hawkBit learns really cluster communication, this code was prototyping of using redis as inter-cluster-communication to distribute events.